.github/workflows/merge-conflict.yml

```diff
@@ -0,0 +1,21 @@
+name: "Check for merge conflicts"
+on:
+  # So that PRs touching the same files as the push are updated
+  push:
+  # So that the `dirtyLabel` is removed if conflicts are resolve
+  # We recommend `pull_request_target` so that github secrets are available.
+  # In `pull_request` we wouldn't be able to change labels of fork PRs
+  pull_request_target:
+    types: [synchronize]
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if PRs are have merge conflicts
+        uses: eps1lon/actions-label-merge-conflict@v2.1.0
+        with:
+          dirtyLabel: "Merge Conflict"
```

Comment:
```suggestion
          dirtyLabel: "PR: Merge Conflict"
```

Reply:
We would need a new label or change the current one.
![Screenshot at 2022-12-17 20-51-32](https://user-images.githubusercontent.com/26622301/208263809-01243b92-f7a6-4bc9-b40d-ad242cfb7416.png)

Reply:
Done.

Reply:
https://github.com/pi-hole/pi-hole/pull/5071 targets master. I didn't feel motivated enough to try `git rebase --onto` for the first time, it was easier to create a new PR.